install.md

```diff
@@ -198,3 +198,96 @@ If you are using OSCAR in the Windows Subsystem for Linux, you will require a br
+</summary>
+Enter the following commands into a file (for example <code>oscar_systemwide_install</code>),
+adjust the paths for the variables <code>julia_for_oscar</code> and <code>central_depot</code>,
+and then run the script in a terminal (with administrator rights).
+{% highlight bash %}
+#!/bin/bash
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.8.5/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# We will store the necessary artifacts in the
+# "architecture dependent location" in Julia's default DEPOT_PATH.
+# First determine this path and create the directory if necessary.
+system_depot=$(${julia_for_oscar} --startup-file=no -e 'println(DEPOT_PATH[2])')
+if ! test -d "${system_depot}/artifacts"; then
+  mkdir -p "${system_depot}/artifacts"
+fi
+
+# Set the Julia variables that control the location of packages.
+# (Do not admit the current user's own depot path.)
+export JULIA_DEPOT_PATH=${central_depot}:${system_depot}
+export JULIA_LOAD_PATH="@:@v#.#:@stdlib:${central_depot}/environments/globalenv"
+
+# Clean the environment, such that the already centrally installed packages
+# get replaced by newer versions if necessary.
+# (This is safer than calling `Pkg.update()` in Julia.)
+# Then let Julia install and precompile the packages.
+${julia_for_oscar} -e 'using Pkg, Artifacts; \
+rm("'${central_depot}'/environments/v" * join(split(string(VERSION), ".")[1:2], ".") * "/Project.toml", force=true)
+Artifacts.with_artifacts_directory("'${system_depot}'/artifacts") do; \
+Pkg.add("Oscar"); \
+Pkg.add("GAP"); \
+Pkg.add("Nemo"); \
+Pkg.add("Hecke"); \
+Pkg.add("Singular"); \
+Pkg.add("Polymake"); \
+Pkg.add("AbstractAlgebra"); \
+Pkg.instantiate(); \
+end; \
+exit();'
+
+# Write a file that will be loaded by Julia on startup.
+echo 'using Pkg; Pkg.activate("'${central_depot}'/environments/v" * join(split(string(VERSION), ".")[1:2], "."))' > ${central_depot}/julia_load_initial.jl
+{% endhighlight %}
+</details>
+
+<details>
+<summary>
+Call Julia such that the system-wide installation of Oscar gets loaded
+when the user enters <code>using Oscar</code> in the Julia session.
+</summary>
+Enter the following commands into a file (for example <code>julia_with_oscar</code>),
+adjust the paths for the variables <code>julia_for_oscar</code> and <code>central_depot</code>,
+and then run the script in a terminal (not with administrator rights).
+{% highlight bash %}
+#!/bin/bash
+
+# Specify the intended version of Julia.
+julia_for_oscar=/users/oscar/julia-1.8.5/bin/julia
+
+# Specify the intended location of the central Oscar installation.
+central_depot=/users/oscar/JULIA_DEPOT
+
+# Admit the user's own depot path and the path with the central installation.
+export JULIA_DEPOT_PATH=$HOME/.julia:${central_depot}
+export JULIA_LOAD_PATH="@:@v#.#:@stdlib:${central_depot}/environments/globalenv"
```

Comment:
Ah and here this trick with "appending" also work:
```suggestion
export JULIA_LOAD_PATH=":@v#.#-oscar"
```
Then I get, after starting julia:
```
julia> LOAD_PATH
4-element Vector{String}:
 "@"
 "@v#.#"
 "@stdlib"
 "@v#.#-oscar"
```

Oh and for the depot I got the following (with `central_depot=/julia_depot`)
```
julia> DEPOT_PATH
4-element Vector{String}:
 "/root/.julia"
 "/usr/local/julia/local/share/julia"
 "/usr/local/julia/share/julia"
 "/julia_depot"
```

Reply:
And this worked just fine:
```
julia> using Singular
Singular.jl, based on
                     SINGULAR                               /
 A Computer Algebra System for Polynomial Computations     /  Singular.jl: 0.18.13
                                                         0<   Singular   : 4.3.2p7
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann   \
FB Mathematik der Universitaet, D-67653 Kaiserslautern      \
```
Even though `Pkg.status()` reports no packages are installed... That's a bit confusing for users, of course (and might make debugging issues a bit harder for them: "which version of AbstractAlgebra are you using?" is harder to answer then, maybe...